packages/fluentui/react-bindings/src/accessibility/getAccessibility.ts

```diff
@@ -36,7 +36,9 @@ const getAccessibility = <Props extends Record<string, any>>(
     // schema is located in @stardust-ui/ability-attributes package.
```

Comment:
As we're touching the place anyway, makes sense to rename that `@stardust-ui/ability-attributes` leftover to `@fluentui/ability-attributes`.